docs/integration/alerts-teams-integration.md

```diff
@@ -0,0 +1,58 @@
+title: Microsoft Teams Alerts Integration
```

Comment:
Can re rename alerts-teams-integration.md to alerts-microsoft-teams-integration.md ?

Reply:
Yup, changed.